config/default.yml

```diff
@@ -198,6 +198,12 @@ PreCommit:
     required_executable: 'grep'
     flags: ['-IHn', '^<<<<<<<\s']
 
+  Pep257:
+    description: 'Analyzing with pep257'
```

Comment:
Since PEP257 specifically deals with docstrings, we should probably say that here.

```
Analyzing docstrings with pep257`
```